File: lista_laco_repeticao/exercicio1.java
import java.util.Scanner;

public class ExibeNumeros {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.println("Este programa exibe todos os números de 1 até o numero escolhido");
        
        System.out.print("Digite um número inteiro: ");
        int numero = scanner.nextInt();
        
        for (int i = 0; i < numero; i++) {
            System.out.println(i + 1);
        }
        
        scanner.close();
    }
}

File: exercicio1.py
print("Este programa exibe 5 numeros ao quadrado")

numeros = []
contador = 0
for i in range (5):
    numero = int(input(f"Digite seu {i+1} número: "))
    numeros.append(numero*numero)
print("Seus numeros ao quadrado: ")
for resultado in numeros:
    print(resultado)